### ISSUE TITLE
Add more modules from torch.nn, match torch.nn structure

### ISSUE THREAD

**Issue by charlesfrye** [charlesfrye/panndas#4]
`module`
- [x] [`Module`](https://pytorch.org/docs/stable/generated/torch.nn.Module.html#torch.nn.Module)

`container`
- [x] [`Sequential`](https://pytorch.org/docs/stable/_modules/torch/nn/modules/container.html#Sequential)

~`conv`: can only support 1D, single-channel~
- [ ] [`Conv1d`](https://pytorch.org/docs/stable/generated/torch.nn.Conv1d.html#torch.nn.Conv1d)
- [ ] [`ConvTranspose1d`](https://pytorch.org/docs/stable/generated/torch.nn.ConvTranspose1d.html#torch.nn.ConvTranspose1d)

~`pooling`: can only support 1D, single channel~
- [ ] [`MaxPool1d`](https://pytorch.org/docs/stable/generated/torch.nn.MaxPool1d.html#torch.nn.MaxPool1d)

`activation`
- [x] [`Mish`](https://pytorch.org/docs/stable/generated/torch.nn.Mish.html#torch.nn.Mish)
- [x] [`Sigmoid`](https://pytorch.org/docs/stable/generated/torch.nn.Sigmoid.html#torch.nn.Sigmoid)
- [x] [`Tanh`](https://pytorch.org/docs/stable/generated/torch.nn.Tanh.html#torch.nn.Tanh)
- [x] [`Softmax`](https://pytorch.org/docs/stable/generated/torch.nn.Softmax.html#torch.nn.Softmax)

`batchnorm`: can only support batch or layer norm
- [x] [`BatchNorm1d`](https://pytorch.org/docs/stable/_modules/torch/nn/modules/batchnorm.html#BatchNorm1d)
- [ ] ~[`LayerNorm`](https://pytorch.org/docs/stable/generated/torch.nn.LayerNorm.html#torch.nn.LayerNorm), and generalize current `LayerMaxNorm` to allow different norms~

`linear`
- [x] [`Identity`](https://pytorch.org/docs/stable/_modules/torch/nn/modules/linear.html#Identity)

`dropout`
- [x] [`AlphaDropout`](https://pytorch.org/docs/stable/generated/torch.nn.AlphaDropout.html#torch.nn.AlphaDropout)
- [x] and special case [`Dropout`](https://pytorch.org/docs/stable/generated/torch.nn.Dropout.html#torch.nn.Dropout)

`loss`
- [ ] `LpLoss`
- [ ] and special cases [`MSELoss`](https://pytorch.org/docs/stable/generated/torch.nn.MSELoss.html#torch.nn.MSELoss) and [`L1Loss`](https://pytorch.org/docs/stable/generated/torch.nn.L1Loss.html#torch.nn.L1Loss)
- [ ] [`CrossEntropyLoss`](https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html#torch.nn.CrossEntropyLoss)

**Comment by charlesfrye** [comment]
decided to only implement functional versions of losses and to skip 1d convs for now
